classes/DirtEventTypes.sc

```diff
@@ -42,7 +42,8 @@ DirtEventTypes {
 				var freq, lag, sustain;
 				var args, midiout, hasGate, midicmd, latency, chan;
 				var sendNRPN, schedmidi, schedmidicmd, donecmd;
-				var hasNote = ~n != \none, midiCommandPending = ~midicmd.notNil;
+				var hasNote = not (~n == \none and: {~note == \none});
```

Comment:
I don't understand why now `~n == \none`, while before it was `~n != \none` ...

Reply:
I got a bit in knots with the nots and ended up sticking a 'not' in front.

Reply:
Replaced with an or, possibly is clearer..